quizapp/src/main/java/billennium/quizapp/service/CandidateService.java

```diff
@@ -21,4 +25,14 @@ public String findByEmail(String email) {
                 .orElseThrow(CandidateException::new);
     }
 
+    public void saveCandidate(String email) {
```

Comment:
jesteśmy w serwisie CandidateService wiec sądze, że nie musi być w nazwie metody `Candidate`. Byłbym bardziej aby dodać do nazwy po czym wyszukujemy zapisujemy czyli w tym przypadku `saveByEmail`

Reply:
ok o ile kiedyś nie zrobi się jakiś miszmasz że jeden servis robi wszystko :) póki co to dobra uwaga

Reply:
hmm masz rację lepiej wygląda candidateService.save od candidateService.saveCandidate